### ISSUE TITLE
Message won't open in a new window

### ISSUE THREAD

**Issue by Pascal666** [tjeb/Mailbox-Alert#26]
When an alert is set up to "Open the message in a new window" on click, a new Thunderbird window opens but no message is displayed.

I am using v0.24 on Thunderbird v68.12.0 (64-bit) on Windows v10.0.18363.1082

**Comment by Pascal666** [comment]
I figured out the problem.  Thunderbird is not processing the filter actions in the order I entered them.  There is a small message at the bottom of the filter to that effect: "Note: Filter actions will be run in a different order."  I told it to move the message to a folder and then trigger the alert.  It was triggering the alert and then moving the message, so by the time I clicked on the alert the message was no longer where Mailbox Alert thought it was.

I can't figure out a way to get it all to work together correctly.
